hippunfold/config/snakebids.yml

```diff
@@ -64,6 +64,19 @@ pybids_inputs:
       - acquisition
       - run
 
+  1mmT1w:
```

Comment:
This part isn't needed, as that would mean we would be grabbing a new bids type called 1mmT1w.. We are just using the same T1w image in our case..